packaging/nuget/nservicebus.host/EndpointConfig.cs

```diff
@@ -3,23 +3,15 @@ namespace rootnamespace
 {
     using NServiceBus;
 
-    /*
-		This class configures this endpoint as a Server. More information about how to configure the NServiceBus host
-		can be found here: http://particular.net/articles/the-nservicebus-host
-	*/
     public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(BusConfiguration configuration)
         {
-            // NServiceBus provides the following durable storage options
-            // To use RavenDB, install-package NServiceBus.RavenDB and then use configuration.UsePersistence<RavenDBPersistence>();
-            // To use SQLServer, install-package NServiceBus.NHibernate and then use configuration.UsePersistence<NHibernatePersistence>();
+            // NServiceBus provides multiple durable storage options
             
             // If you don't need a durable storage you can also use, configuration.UsePersistence<InMemoryPersistence>();
```

Comment:
Remove the new lines? (https://github.com/Particular/NServiceBus.Host/blob/fixInjectedCode/packaging/nuget/nservicebus.host/EndpointConfig.cs , line 11 +13)